agent/lib/utils.js

```diff
@@ -223,3 +223,26 @@ module.exports.changes = function (last, current, compare, unchanged, stringify)
         }
     }
 };
+
+const bytes_pattern = /^(\d*)\s*(KB|MB|GB)?$/i;
+module.exports.parse_bytes = function (s) {
```

Comment:
I can't see this being used anywhere, is it still needed?

Reply:
It is not, no.